packages/kotti-ui/source/kotti-filters/components/FilterRow.vue

```diff
@@ -146,39 +163,25 @@ export default defineComponent<{
 			}
 		})
 		const valuePrefix = computed<KottiFieldNumber.Props['prefix']>(() =>
-			props.column?.type === KottiFilters.FilterType.CURRENCY ||
 			props.column?.type === KottiFilters.FilterType.FLOAT
 				? props.column.prefix
 				: null,
 		)
 		const valueStep = computed<KottiFieldNumber.Props['step']>(() => {
 			switch (props.column?.type) {
-				case KottiFilters.FilterType.CURRENCY:
 				case KottiFilters.FilterType.FLOAT:
 					return props.column.step
 				case KottiFilters.FilterType.INTEGER:
-				// fall through
-				default:
 					return 1
+				default:
+					return null
```

Comment:
can you type cast all returns from
`valueStep`, `valuePrefix`, `valueSuffix`

this will make us avoid this bug in the future when someone copies the implementation for "additionalProps"

so they can return `null` when the prop is not relevant

Reply:
Now `prefix`, `suffix`, `decimalPlaces`, `step`, `currency` and `options` will default to `undefined` for types that don't support them.